hack/ci-android-e2e.sh

```
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+# Runs the Maestro e2e suite against an already-booted Android emulator.
+# This must be a single script (not inline workflow lines) because
+# reactivecircus/android-emulator-runner executes each line of `script:`
+# in a separate shell, so environment variables don't survive between lines.
+set -euo pipefail
+
+. /tmp/e2e-env.sh
```

Shellcheck (0.11.0)

[info] Not following: /tmp/e2e-env.sh was not specified as input (see shellcheck -x).

(SC1091)